eras/allegra/impl/cddl-files/allegra.cddl

```diff
@@ -305,27 +310,34 @@ script_any = (2, [* native_script])
 
 script_n_of_k = (3, n : int64, [* native_script])
 
-int64 = -9223372036854775808 .. 9223372036854775807
+int64 = min_int64 .. max_int64
+
+min_int64 = -9223372036854775808
+
+max_int64 = 9223372036854775807
 
 invalid_before = (4, uint)
 
 invalid_hereafter = (5, uint)
 
 bootstrap_witness = 
-  [ public_key : vkey          
-  , signature  : signature     
-  , chain_code : bytes .size 32
-  , attributes : bytes         
+  [ public_key : vkey     
+  , signature  : signature
+  , chain_code : bytes32  
+  , attributes : bytes    
   ]
 
 
-transaction_index = uint .size 2
+transaction_ix = uint .size 2
 
 auxiliary_data = 
-  metadata                                                                              
-  / [transaction_metadata : metadata, auxiliary_scripts : auxiliary_scripts]
+  shelley_auxiliary_data                                                                     
```

Comment:
This is incorrect. It just so happens that in Shelley `auxiliary_data` is set to just metadata. But `metadata` is it own concept: `Map Word64 Metadatum`
```suggestion
  metadata                                                                     
```